src/web/components/KeyPairs/KeyPairModel.tsx

```diff
@@ -15,12 +14,12 @@ export type KeyPairModel = {
 export const mapKeyPairDTOToModel = (dto: KeyPairDTO): KeyPairModel => {
   const model: KeyPairModel = {
     contact: dto.contact as string | undefined,
-    createdString: new Date(dto.created * 1000).toUTCString(),
-    created: new Date(dto.created * 1000),
+    createdString: new Date(dto.created).toISOString().split('T')[0],
```

Comment:
nit: I wonder if we could use .toLocaleDateString() instead of `.toISOString().split('T')[0]`?

Reply:
done!